yarn.lock

```diff
@@ -2677,6 +2677,13 @@ ast-types@^0.13.2:
   dependencies:
     tslib "^2.0.1"
 
+ast-types@^0.13.2:
```

Comment:
[mid]: why those dependencies have been added?

Reply:
Not sure, can't repro it now, deleted 👍